backend/src/test/java/com/redhat/cloud/notifications/routers/EventServiceTest.java

```diff
@@ -438,13 +456,14 @@ void testAllQueryParams() {
                      * Account: DEFAULT_ACCOUNT_ID
                      * Request: EMAIL_SUBSCRIPTION endpoints and invocation failed
                      */
-                    return getEventLogPage(defaultIdentityHeader, null, null, null, null, null, Set.of(EMAIL_SUBSCRIPTION), Set.of(FALSE), null, null, null);
+                    return getEventLogPage(defaultIdentityHeader, null, null, null, null, null, Set.of(EMAIL_SUBSCRIPTION), Set.of(FALSE), null, null, null, false);
                 }))
                 .emitOn(MutinyHelper.executor(vertx.getOrCreateContext()))
                 .invoke(page -> {
                     assertEquals(1, page.getMeta().getCount());
                     assertEquals(1, page.getData().size());
                     assertSameEvent(page.getData().get(0), model.events.get(0), model.notificationHistories.get(0), model.notificationHistories.get(1));
+                    assertNull(page.getData().get(0).getPayload());
```

Comment:
Is there any test when this is not null?

Reply:
Yes, it's well hidden 😄